test/secrets_test.rb

```diff
@@ -0,0 +1,53 @@
+require 'test_helper'
+
+class SecretsTest < Minitest::Test
+  def test_equals_ignore_case
+    sample_config = {
+      "matcher"=>"equals-ignore-case", 
+      "list"=>["key"]
+    }
+    
+    subject = Instana::Secrets.new
+    assert_equal 'http://example.com/?kEy=<redacted>', subject.remove_from_query('http://example.com/?kEy=abcde', sample_config)  
```

Comment:
Can we also add other query params that are not supposed to be filtered here and below?